pom.xml

```
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1850-SNAPSHOT</version>
```

```suggestion
    <version>1.0-SNAPSHOT</version>
```
I'm reverting this back to `1.0-SNAPSHOT`, as it passes in Jenkins! Thank you! :)